src/main/java/Duke.java

```diff
@@ -1,10 +1,223 @@
+import Task.Task;
+import Task.Deadline;
+import Task.Todo;
+import Task.Event;
+
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class Duke {
+    static int MAX_CAPACITY = 100;
```

Comment:
I like how you avoid "magic numbers" and define them as a variable. I see this in other place of your codes as well.